zgw_consumers/admin.py

```diff
@@ -16,6 +17,14 @@ class ServiceAdmin(admin.ModelAdmin):
     list_display = ("label", "api_type", "api_root", "nlx", "auth_type")
     list_filter = ("api_type", "auth_type")
     search_fields = ("label", "api_root", "nlx", "uuid")
+    readonly_fields = ("get_connection_check",)
+
+    @admin.display(description=_("Connection check status code"))
+    def get_connection_check(self, obj):
+        if obj.pk is None:
+            return "n/a"
```

Comment:
```suggestion
            return _("n/a")
```
mark strings as translatable ;) so we can display "n.v.t." in Dutch